cluster/gce/gci/configure-helper.sh

```diff
@@ -1878,10 +1889,15 @@ function start-etcd-servers {
   if [[ -e /etc/init.d/etcd ]]; then
     rm -f /etc/init.d/etcd
   fi
-  prepare-log-file /var/log/etcd.log
+  if [[ -n "${ETCD_RUNASUSER:-}" && -n "${ETCD_RUNASGROUP:-}" ]]; then
+    prepare-log-file /var/log/etcd.log  ${ETCD_RUNASUSER} ${ETCD_RUNASGROUP}
```

Comment:
You only need to pass the user and not the group see: https://github.com/kubernetes/kubernetes/blob/816bdd3011d093b5e60dd4ca3be10f87b315826e/cluster/gce/gci/configure-helper.sh#L2029

Reply:
Fixed.